R/dockerfile.R

```diff
@@ -27,38 +19,130 @@
 #' dockerfile()
 #'
 #' @import futile.logger
-dockerfile <- function(from = sessionInfo(), to = paste0(getwd(), "/", "Dockerfile")) {
-  flog.debug("Creating a new Dockerfile from %s to %s", from, to)
+dockerfile <- function(from = utils::sessionInfo(),env = NULL, maintainer = NULL, r_version = paste(R.Version()$major, R.Version()$minor, sep="."), image = NULL, context = NA_character_) {
```

Comment:
- Use empty list as default for `env` or even a non-empty one, e.g. `env = list("generator" = paste("containerit", packageVersion("containerit))`. Should be easier to implement than checking for NULL, and tells the user what to put in.
- Use a default image, i.e. `image = default_image()`, which simply does `fromImage(paste(R.Version()$major, R.Version()$minor, sep="."))`